### PR TITLE
[Feature] add min, max, 80th & 20th percentiles as HRV features

### PR DESCRIPTION
**Pull request by danibene**
Add the minimum, maximum, 20th percentile, and 80th percentile NN as features in the hrv_time function.

- [x] I have read the [CONTRIBUTING](https://github.com/neuropsychology/NeuroKit/blob/master/.github/CONTRIBUTING.rst#structure-and-code) file.
- [x] My PR is targeted at the **dev branch** (and not towards the master branch).
- [x] I ran the [CODE CHECKS](https://github.com/neuropsychology/NeuroKit/blob/master/.github/CONTRIBUTING.rst#run-code-checks) on the files I added or modified and fixed the errors.
- [x] I have added the newly added features to **News.rst** (if applicable)

### CODE REVIEW

File: neurokit2/hrv/hrv_time.py [hunk 1]
```diff
@@ -50,8 +50,12 @@ def hrv_time(peaks, sampling_rate=1000, show=False, **kwargs):
         - **HCVNN**: The median absolute deviation of the RR intervals (MadNN) divided by the median
         of the absolute differences of their successive differences (MedianNN).
         - **IQRNN**: The interquartile range (IQR) of the RR intervals.
+        - **Prc20NN**: The 20th percentile of the RR intervals.
```
Comment (DominiqueMakowski) [inline neurokit2/hrv/hrv_time.py:53]: @danibene Great, thanks, very clean PR :)

It's mostly good, but could you perhaps add one or two references in which they use these indices? You can add the full ref in the Reference section and the short form (refauthor, 1998) after the index (in the future we could probably add that to all of the indices but that will be WIP)

**Comment by DominiqueMakowski** [comment]
Also, consider adding yourself to the [contributor](https://github.com/neuropsychology/NeuroKit/blob/master/AUTHORS.rst#contributors) list if you want

File: neurokit2/hrv/hrv_time.py (hunk 1 above)
Comment (danibene) [reply neurokit2/hrv/hrv_time.py]: Thank you :)

I added two references for each but please feel free to add/remove/edit.

**Comment by DominiqueMakowski** [comment]
Looks great, thanks again @danibene !! will merge as soon as the checks pass

**Pull request merged**